graylog2-web-interface/src/pages/ShowContentPackPage.jsx

```diff
@@ -47,16 +56,18 @@ const ShowContentPackPage = createReactClass({
     if (window.confirm('You are about to delete this content pack, are you sure?')) {
       ContentPacksActions.deleteRev(contentPackId, revision).then(() => {
         UserNotification.success('Content Pack deleted successfully.', 'Success');
```

Comment:
I think this feedback message is somehow outdated, right? Otherwise I don't understand why we delete a content pack and then try to fetch it.

Reply:
Because we delete a content pack revision. The content pack remains. We refetch the content pack to show the updated list of revisions. But if you delete the last content pack revision we want to redirect to the overview.

Reply:
Alright, then the message should say something like "Content Pack revision deleted successfully". I would be confused if I see the original message when I'm deleting some old version of a Content Pack.

Reply:
Oh, I saw you changed it already. Sorry about the noise 😄